src/pages/Admin/Apps/Deploy/index.jsx

```diff
@@ -293,7 +293,8 @@ export default class AppDeploy extends Component {
     const { isKubernetes } = appDeployStore;
     const { isLoading } = appStore;
     const title = `${t('Deploy')} ${appDetail.name}`;
-    const { isNormal } = user;
+    const { isNormal, isDev } = user;
+    const linkPath = isDev ? 'My Apps>Test>Deploy' : 'Store>All Apps>Deploy';
```

Comment:
hard code link path ? `linkPath` is label or url ?

Reply:
In page linkPath is string, in component change to corresponding link by pathToLink。

Reply:
I think you can abstract your `NavLink` component to `Breadcrumb` component, and make it more reusable